app/src/main/res/values-fr/strings.xml

```diff
@@ -234,6 +241,11 @@
   <string name="miband_prefs_reserve_alarm_calendar">Alarmes à réserver pour événements futurs</string>
   <string name="miband_prefs_hr_sleep_detection">Utiliser le capteur cardiaque pour améliorer la précision du sommeil</string>
   <string name="miband_prefs_device_time_offset_hours">La compensation de temps en heure (pour détecter le sommeil de travailleurs en rotation, par exemple)</string>
+  <string name="miband2_prefs_dateformat">Mi band 2 : format de l\'heure</string>
+  <string name="dateformat_time">Heure seule</string>
+  <string name="dateformat_date_time"><![CDATA[Heure & date]]></string>
+  <string name="mi2_prefs_activate_display_on_lift">Alumer quand le poignet se lève</string>
```

Comment:
Just fixes things with some more commits and push to your branch, they will appear here :)

> Once you've created a pull request, you can push commits from your topic branch to add them to your existing pull request. These commits will appear in chronological order within your pull request and the changes will be visible in the "Files changed" tab.
>
> https://help.github.com/articles/about-pull-requests/